server/graphql/common/expenses.ts

```diff
@@ -244,6 +245,13 @@ export const canPayExpense = async (req: express.Request, expense: typeof models
   } else if (!canUseFeature(req.remoteUser, FEATURE.USE_EXPENSES)) {
     return false;
   } else {
+    const host = await req.loaders.Expense.host.load(expense.id);
+    if (hasOptedInForFeature(host, FEATURE.APPROVERS_CANNOT_PAY_EXPENSES)) {
+      const approver = await expense.getApproverUser();
```

Comment:
Not requested, but I'd be tempted to put this one in a loader as well. The `Activities` table is very big, querying it is always a bit risky (even tho it's well indexed on `ExpenseId`).

Reply:
I'll punt this one because I'm planning to refactor how we track approvals in https://github.com/opencollective/opencollective/issues/3081.